docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -121,26 +127,32 @@ endif::[]
 
 Before you use {product-connectors}, you must complete the following prerequisites: 
 
-* Determine which {openshift} environment to use for deploying your {product-connectors} instances.
+* Determine which {openshift} environment to use for your _{connectors} namespace_. The {connectors} namespace is where your {product-connectors} instances will be deployed.
```

Comment:
Now that you have the description of "namespace" in this section, perhaps remove the text "The namespace is the deployment space that hosts your Connectors instances." from both the "Creating a Connectors instance for a data source" and "Creating a Connectors instance for a data sink" sections?

Reply:
Resolving as Done :)